src/ui/components/Timeline/PreviewMarkers.tsx

```diff
@@ -75,25 +75,38 @@ function PreviewMarkersSuspends({
     index => lineNumber - hitCounts[index][0]
   );
 
-  const hitCountsForLine = hitCounts[hitCountsForLineIndex];
+  let firstColumnHitCounts = undefined;
   let firstColumnWithHitCounts = undefined;
+  const hitCountsForLine = hitCounts[hitCountsForLineIndex];
   if (hitCountsForLine && hitCountsForLine[0] === lineNumber) {
+    firstColumnHitCounts = hitCountsForLine[1].count;
     firstColumnWithHitCounts = hitCountsForLine[1].firstBreakableColumnIndex;
   }
 
-  const [hitPoints, hitPointStatus] = hitPointsForLocationCache.read(
-    replayClient,
-    {
-      begin: focusRange ? focusRange.begin.point : "0",
-      end: focusRange ? focusRange.end.point : endpoint,
-    },
-    {
-      column: firstColumnWithHitCounts ?? 0,
-      line: lineNumber,
-      sourceId,
-    },
-    null
-  );
+  let hitPoints: TimeStampedPoint[] = [];
+  let hitPointStatus: HitPointStatus | null = null;
+  if (
+    firstColumnWithHitCounts != null &&
+    firstColumnHitCounts != null &&
+    firstColumnHitCounts > 0
+  ) {
+    const tuple = hitPointsForLocationCache.read(
+      replayClient,
+      {
+        begin: focusRange ? focusRange.begin.point : "0",
+        end: focusRange ? focusRange.end.point : endpoint,
+      },
+      {
+        column: firstColumnWithHitCounts ?? 0,
```

Comment:
`?? 0` shouldn't be necessary anymore, right?

Reply:
Yeah, that's the part they specifically want us to avoid.